Dockerfile

```diff
@@ -21,6 +21,15 @@ LABEL maintainer="Parseable Team <hi@parseable.io>"
 LABEL org.opencontainers.image.vendor="Parseable Inc"
 LABEL org.opencontainers.image.licenses="AGPL-3.0"
 
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+    cmake \
+    librdkafka-dev \
+    ca-certificates \
+    libsasl2-dev \
+    libssl-dev && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Why do we need these?

Reply:
These system dependencies are needed for [rust-rdkafka](https://github.com/fede1024/rust-rdkafka) which binds to [librdkafka](https://github.com/confluentinc/librdkafka) ([docs](https://docs.confluent.io/platform/current/clients/librdkafka/html/index.html)):

- `librdkafka-dev`: Core C lib for rdkafka bindings
- `cmake`: For building native dependencies
- `libsasl2-dev` & `libssl-dev`: For TLS and auth support (client certs, SASL/PLAIN, SASL/GSSAPI)
- `ca-certificates`: SSL cert verification

Unfortunately, Pure Rust clients like [`kafka-rust`](https://github.com/kafka-rust/kafka-rust) are unmaintained, making rdkafka + native dependencies the viable option for production use.

Reply:
https://github.com/parseablehq/parseable/pull/1047/files#diff-0b9a8b39060176edfa1d676b71bae55517918de9ef34cd05d3f532559ef9c7e8R764